contrib/dracut/90zfs/module-setup.sh.in

```diff
@@ -52,6 +52,8 @@ install() {
 	fi
 	dracut_install @sbindir@/mount.zfs
 	dracut_install @udevdir@/vdev_id
+	dracut_install awk
+	dracut_install head
```

Comment:
Don't we also need `sed`, `cat`, `ls` and `udevadm`? Are these included by default?

Also, i think we could use `seq 1q` instead of `head -1` and avoid including `head`.